flasksrc/__init__.py

```diff
@@ -1,8 +1,8 @@
 import os
-
 from flask import Flask
 from flask_restful import Resource, Api
 from flask_jwt_extended import JWTManager
+from flask_apispec import FlaskApiSpec
 
 
 class HelloReclass(Resource):
```

Comment:
Should we create a better name for this or have a better response if this is a status?

Reply:
Looks like the Hello World for the app/easy heartbeat/liveness check. We could delete in another refactor if needed.